### PR TITLE
add atlantis DNS record

### PR DESCRIPTION
**Pull request by upodroid**
Required for #7520

### CODE REVIEW

File: dns/zone-configs/k8s.io._0_base.yaml [hunk 1]
```diff
@@ -274,6 +274,11 @@ oauth2-proxy:
     value: 34.66.218.218
   - type: AAAA
     value: "2600:1900:4000:627b:8000::"
+atlantis:
+  - type: A
```
Comment (BenTheElder) [inline dns/zone-configs/k8s.io._0_base.yaml:278]: This is in cluster AAA?

Comment (BenTheElder) [reply dns/zone-configs/k8s.io._0_base.yaml:278]: Ack- Sorry I meant to come back and wound up on the kubebuilder thread through 😴 (not the one in the channel, direct message with me and Fede)

**Comment by upodroid** [comment]
Thanks

/hold cancel

**Pull request merged**